src/main/kotlin/no/nav/syfo/sykmeldingstatus/api/SykmeldingBekreftApi.kt

```diff
@@ -19,12 +19,12 @@ fun Route.registerSykmeldingBekreftApi(sykmeldingStatusService: SykmeldingStatus
         val token = call.request.headers["Authorization"]!!
         val principal: JWTPrincipal = call.authentication.principal()!!
         val fnr = principal.payload.subject
-        val sykmeldingBekreftEventDTO = call.receive<SykmeldingBekreftEventUserDTO>()
+        val sykmeldingBekreftEventDTO = call.receiveOrNull<SykmeldingBekreftEventUserDTO>()
```

Comment:
Ah, sorry, jeg tenkte ikke på at det kanskje ble litt teit for avviste sykmeldinger det her.. Fungerte det å gjøre det sånn? Vi har ganske gode enhetstester for dette apiet, så du kan antageligvis legge til en test der for å se om det virker eller ikke uten å måtte vente på bygg og deploy :)

Reply:
God idé! Ja, jeg kan lage en enhetstest på det :)

Reply:
Det funket ikke! :D Bør funke nå.